### PR TITLE
Expand $aDemoSuite and revise code that launches demo functions

### PR DESCRIPTION
**Pull request by Danp2**
See #155

### CODE REVIEW

**Comment by Danp2** [comment]
@mlipok Please review these changes

**Comment by mlipok** [comment]
I was trying to add code to yours mod
https://github.com/Danp2/WebDriver/pull/158

But something went wrong.
I was not intended to do separate PR but submit to this particular one.

File: wd_demo.au3 [hunk 1]
```diff
@@ -19,21 +19,21 @@ Global Const $aBrowsers[][2] = _
 		["MSEdge", SetupEdge] _
 		]
 
-Global $aDemoSuite[][2] = _
+Global $aDemoSuite[][3] = _
```
Comment (mlipok) [inline wd_demo.au3:22]: Please add comments
```AutoIt
Global $aDemoSuite[][3] = _ ; COL0 - FunctionName, COL1 - Selected at start, COL2 - Pass browser name as parameter to callad FunctionName
```

**Review by mlipok — APPROVED** [review]
Looking good. Only a request for comment on the $aDemoSuite[][3] array

**Pull request merged**